sycl/ReleaseNotes.md

```diff
@@ -13,6 +13,7 @@ codes in the same translation unit [31608c24] [f5126d28]
 `-fsycl-targets` [5e86a41d]
 
 ### SYCL Library
+- Made use of immediate command lists the default on Intel® Data Center GPU Max Series on Linux [9e5b2ed1] (https://www.intel.com/content/www/us/en/developer/articles/guide/level-zero-immediate-command-lists.html)
```

Comment:
@rdeodhar : since imm lists are only for the L0 backend, should that be mentioned? Something like:

"Made use of immediate command lists in the L0 GPU backend the default ..."

Reply:
This item is just for adding new documentation, so suggest the following template: "Added a performance guide for XXX"

Reply:
The item is not about new documentation. It is about an implementation change that makes use of immediate command lists the default when using L0. No additions were made to the in-tree documentation about this implementation change. Instead, a note has been placed on an Intel web-site and the RN links to that note.

Reply:
And yes, I have clarified that it applies only to L0.